### ISSUE TITLE
[Bug] [a11y]Onboarding screen-The text contrast ratio needs to be higher

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9579]
Prerequisites:
- Have Google Accessibility Scanner installed.
- Have a clean Firefox profile.

## Steps to reproduce
1. Open app and inspect the screen elements with the scanner.
### Expected behavior
There are no suggestions to make any changes.

### Actual behavior
**Section: Welcome to Firefox Nightly**
- org.mozilla.fenix.nightly:id/header_text
The item's text contrast ratio is 3.39. This ratio is based on an estimated foreground color of #A36AFA and an estimated background color of #FBFBFE. Consider using colors that result in a contrast ratio greater than 4.50.

**Section: Protect yourself**
- org.mozilla.fenix.nightly:id/protection_standard_description
The item's text contrast ratio is 3.61. This ratio is based on an estimated foreground color of #737373 and an estimated background color of #E0E0E6. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text. 

### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Nightly 4/1 (Build #20920607)

**Screenshots:**
**- org.mozilla.fenix.nightly:id/header_text**
![20200401_121716](https://user-images.githubusercontent.com/49859863/78120666-037dd400-7413-11ea-8e4a-a75e960400d9.jpg)

**- org.mozilla.fenix.nightly:id/protection_standard_description**
![20200401_121642](https://user-images.githubusercontent.com/49859863/78120659-0082e380-7413-11ea-9058-02f2d828c65e.jpg)

**Comment by mcarare** [comment]
Can UX provide colors that pass the contrast ratio test?

**Comment by liuche** [comment]
@apbitner since Bram is on PTO, could you suggest a higher contrast color? Whoever picks this up can ping on slack please.

**Comment by apbitner** [comment]
Per #7606 all onboarding cards should have a white background, which would not have this contrast issue. Pinging @AmyYLee to confirm.

**Comment by AmyYLee** [comment]
> Per #7606 all onboarding cards should have a white background, which would not have this contrast issue. Pinging @AmyYLee to confirm.

That's correct.

**Comment by mcarare** [comment]
@apbitner you are right, and the changes are already implemented. I have rescanned onboarding screen and there are no contrast issues left. Thanks for the feedback!
